<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvatarAppearanceDialog</class>
 <widget class="QDialog" name="AvatarAppearanceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-107</y>
        <width>485</width>
        <height>1550</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>

       
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         
         
         
       <item>
        <widget class="QRadioButton" name="useFullAvatar">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>Use single avatar with Body and Head</string>
         </property>
        </widget>
       </item>



       <item>
        <layout class="QVBoxLayout" name="verticalLayout_fullAvatar">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         
         
         <item>
          <widget class="QLabel" name="fullAvatarNameLabel">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="text">
            <string>Full Avatar</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="buddy">
            <cstring>fullAvatarURLEdit</cstring>
           </property>
          </widget>
         </item>

         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_fullAvatar">

           <item>
            <widget class="QLineEdit" name="fullAvatarURLEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>

           <item>
            <spacer name="horizontalSpacer_fullAvatar">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>5</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>

           <item>
            <widget class="QPushButton" name="buttonBrowseFullAvatar">
             <property name="font">
              <font>
               <family>Arial</family>
              </font>
             </property>
             <property name="text">
              <string>Browse</string>
             </property>
             <property name="iconSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           
           
           
          </layout>
         </item>
         
         
         
        </layout>
        
        
        
        
       </item>

       <item>
        <widget class="QRadioButton" name="useSeparateBodyAndHead">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>Use separate Body and Head avatar files</string>
         </property>
        </widget>
       </item>


       <item>
        <layout class="QVBoxLayout" name="verticalLayout_separateParts_head">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>



         <item>
          <widget class="QLabel" name="headNameLabel">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="text">
            <string>Head</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>

         </item>
         
         
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="faceURLEdit"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>5</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="buttonBrowseHead">
             <property name="font">
              <font>
               <family>Arial</family>
              </font>
             </property>
             <property name="text">
              <string>Browse</string>
             </property>
             <property name="iconSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         
         
        </layout>
       </item>

       <item>
        <layout class="QVBoxLayout" name="verticalLayout_separateParts_body">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         
         <item>
          <widget class="QLabel" name="bodyNameLabel">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="text">
            <string>Body</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="buddy">
            <cstring>skeletonURLEdit</cstring>
           </property>
          </widget>

         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_skeletonURL">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="skeletonURLEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>5</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="buttonBrowseBody">
             <property name="font">
              <font>
               <family>Arial</family>
              </font>
             </property>
             <property name="text">
              <string>Browse</string>
             </property>
             <property name="iconSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>




        </layout>
       </item>


      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonsPanel">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <layout class="QHBoxLayout" name="buttonsHBox_2">
        <item>
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="defaultButton">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
</ui>
